code/modules/food_and_drink/pies.dm

```diff
@@ -9,12 +9,44 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/pie)
 	var/slicetype = /obj/item/reagent_containers/food/snacks/pieslice
 	var/splat = 0 // for thrown pies
 	food_effects = list("food_refreshed","food_cold")
+	///In the case of a thrown splattered pie, minimum amount of time we remain visually stuck on someone's face.
+	var/min_stuck_time = 5 SECONDS
+	///In the case of a thrown splattered pie, maximum amount of time we remain visually stuck on someone's face.
+	var/max_stuck_time = 10 SECONDS
 
 	throw_impact(atom/hit_atom, datum/thrown_thing/thr)
 		if (ismob(hit_atom) && src.splat)
 			var/mob/M = hit_atom
-			src.visible_message("<span class='alert'>[src] splats in [M]'s face!</span>")
+			var/mob/thrower = usr
 			playsound(src, 'sound/impact_sounds/Slimy_Splat_1.ogg', 100, 1)
+			if (thrower.mind?.assigned_role == "Clown")
+				if (ishuman(M) && (prob(50) || M.mind?.assigned_role == "Captain"))
```

Comment:
There should be something here to check if the target has a pie on them because you can keep stacking them